tests/filecheck/dialects/onnx/onnx_ops.mlir

```diff
@@ -0,0 +1,10 @@
+// RUN: XDSL_ROUNDTRIP
+
+func.func @add(%arg0: tensor<1x2x6xf32>, %arg1: tensor<1x2x6xf32>) ->  tensor<1x2x6xf32> {
+  %0 = "onnx.Add"(%arg0, %arg1) {onnx_node_name = "/model.1/Add"} : (tensor<1x2x6xf32>, tensor<1x2x6xf32>) -> tensor<1x2x6xf32>
```

Comment:
Could you please add tests for the other ops, also?

Reply:
Done